2019-12-27-what-is-http-messages.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Learning http messages"
+date: "2019-12-27 11:40:00 UTC"
+daft: true
```

Comment:
This is actually a feature from my blog and doesn't automatically work. It's also misspelled